metacat-index/src/main/java/edu/ucsb/nceas/metacat/index/MetacatIndexServlet.java

```diff
@@ -46,37 +46,47 @@ public class MetacatIndexServlet extends HttpServlet {
     // see http://www.docjar.com/docs/api/org/springframework/context/support/FileSystemXmlApplicationContext.html
     //private static final String FILEPREFIX = "file:";
     
-	private static Log log = LogFactory.getLog(MetacatIndexServlet.class);
+    private static Log log = LogFactory.getLog(MetacatIndexServlet.class);
 
     /**
-     * Initialize the servlet 
+     * Initialize the servlet
      */
     public void init(ServletConfig config) throws ServletException {
         //System.out.println("++++++++++++++++++++++++------------------- start the servlet");
-    	//initializeSharedConfiguration(config);
-    	// initialize the application using the configured application-context
+        //initializeSharedConfiguration(config);
+        // initialize the application using the configured application-context
         //URL url = getClass().getResource("/index-processor-context.xml");
         //find the sibling metacat.properties file
-        String metacatPropertiesFilePath = config.getServletContext().getInitParameter("metacat.properties.path");
+        String metacatPropertiesFilePath =
+            config.getServletContext().getInitParameter("metacat.properties.path");
         File contextDeploymentDir = new File(config.getServletContext().getRealPath("/"));
-        String fullMetacatPropertiesFilePath = contextDeploymentDir.getParent()  + metacatPropertiesFilePath;
+        String fullMetacatDefaultPropertiesFilePath =
+            contextDeploymentDir.getParent() + metacatPropertiesFilePath;
+        String fullMetacatSitePropertiesFilePath =
```

Comment:
Can get the site properties location from the metacat.properties file. Will refactor